packages/tests-e2e/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@wordpress/tests-e2e",
```

Comment:
One other thought on this, I was wondering if tests-e2e might be too much of a generic name. I'm not sure if we'd consider having more than just gutenberg/block-editor tests in this package.

Reply:
Agree that we should have more granular control on what tests can be run. I expect this to grow past the editor only. At some point we'd be able to run phase2 tests, core editor tests, ... separately. It could be just . `import { editorTests } from "@wordpress/tests-e2e"`.

I don't have a strong opinion or a perfect solution at the moment though, this is probably something we'd have to iterate on.

Reply:
Jest support projects, so we can split it into smaller groups using this feature:

https://jestjs.io/docs/en/configuration#projects-array-string-projectconfig